src/fiskaltrust.Launcher/Commands/Common.cs

```diff
@@ -25,7 +25,11 @@ public CommonCommand(string name, bool addCliOnlyParameters = true) : base(name)
             AddOption(new Option<string?>("--access-token"));
             AddOption(new Option<bool>("--sandbox"));
             AddOption(new Option<string?>("--log-folder"));
-            AddOption(new Option<LogLevel?>("--log-level"));
+
+            var logLevelOption = new Option<LogLevel?>("--log-level", "Set the log level of the application.");
+            logLevelOption.AddAlias("-v");
+            logLevelOption.AddAlias("--verbosity");
+            AddOption(logLevelOption);
```

Comment:
```suggestion
AddOption(new Option<LogLevel?>(new[] { "--log-level", "-v", "--verbosity" }, "Sets the verbosity level of the logging."));
```

Would it also work like that here with the nicer inline syntax?

Reply:
Sure thing I missed of that : ) thanks Paul